src/components/DsfrFileUpload/DsfrFileUpload.vue

```diff
@@ -41,6 +41,11 @@ const onChange = ($event: InputEvent) => {
       :for="id"
     >
       {{ label }}
+      <span
+        v-if="$attrs.required"
```

Comment:
Je pense qu’il faut mettre `'required' in $attrs` pour avoir le résultat attendu. Je dois d’ailleurs corriger ce bug dans DsfrInput aussi.